mx.jruby/mx_jruby.py

```diff
@@ -63,41 +63,36 @@ def getResults(self):
 
 # Commands
 
-def extractArguments(cli_args):
+def extractArguments(args):
     vmArgs = []
     rubyArgs = []
     classpath = []
     print_command = False
 
-    jruby_opts = os.environ.get('JRUBY_OPTS')
-    if jruby_opts:
-        jruby_opts = jruby_opts.split(' ')
```

Comment:
This probably needs to handle RUBYOPT and TRUFFLERUBYOPT, unless the Main already take care of that and Java options are disallowed in these env vars.
OTOH, I think this launcher is only used when invoking `mx ruby` directly so it should not cause failures in CI.
It simulates how GraalVM runs, but maybe we should just remove it and test with the full GraalVM when needed.